guides/release-notes.md

```diff
@@ -7,6 +7,15 @@ We love hearing your feedback! You can help us improve the docs by reporting any
 ![Was this helpful, yes/no buttons](../media/feedback.png)
 
 ---
+### New: Blocks Panel Builder Elements
```

Comment:
```suggestion
### New: Wix Blocks Panel Color and Font Picker Elements
```

- "Blocks" on its own isn't a term.
- Be specific about the elements being released